### DISCUSSION TITLE
Massage markdown-links for GitLab platform - like Github

### DISCUSSION THREAD

**Discussion by MaxWinterstein** [renovatebot/renovate#29894] [Suggest an Idea]
Currently there is a lot of massaging in the PR Description going on, like rewriting github.com references to togithub.com for backlink prevention and more.

The GitLab platform is missing this functionality which leads to e.g. wrong backlinks within the same repo:
When the changelog contains simple relative reference like `!123` the Merge Request will ping his neighbour Merge Request with id `123`, even it is years old.

(Originally posted as discussion https://github.com/renovatebot/renovate/discussions/29892)

**Comment by rarkins** [comment]
Converted to issue: https://github.com/renovatebot/renovate/issues/29934
